### ISSUE TITLE
Test failure with go 1.10

### ISSUE THREAD

**Issue by Ikke** [junegunn/fzf#1236]
- Category
    - [ ] fzf binary
    - [ ] fzf-tmux script
    - [ ] Key bindings
    - [ ] Completion
    - [ ] Vim
    - [ ] Neovim
    - [x] Etc.
- OS
    - [x] Linux
    - [ ] Mac OS X
    - [ ] Windows
    - [ ] Windows Subsystem for Linux
    - [ ] Etc.

Since Go 1.10, make test is failing due to issues with Errorf, for example:

> gopath/src/github.com/junegunn/fzf/src/ansi_test.go:29: Errorf format %s has arg []rune(output) of wrong type []rune

And a lot more of these.

Since Go 1.10, vetting has been enabled:

> The go test command now automatically runs go vet on the package being tested, to identify significant problems before running the test. Any such problems are treated like build errors and prevent execution of the test.

I can make the tests pass by adding -vet=off

Applying this patch lets the test pass, but to me it looks like this issue should be properly solved

```diff
diff --git a/Makefile b/Makefile
index 5f68100828..123bfe9974 100644
--- a/Makefile
+++ b/Makefile
@@ -102,7 +102,7 @@ vendor: $(GLIDE_YAML)
        touch $@

 test: $(SOURCES) vendor
-       SHELL=/bin/sh GOOS= go test -v -tags "$(TAGS)" \
+       SHELL=/bin/sh GOOS= go test -v -tags "$(TAGS)" -vet=off\
                                github.com/junegunn/fzf/src \
                                github.com/junegunn/fzf/src/algo \
                                github.com/junegunn/fzf/src/tui \
```

ps. Not sure if really a problem, but this message also shows up now:

> loadinternal: cannot find runtime/cgo

**Comment by junegunn** [comment]
A fix to the problem was proposed in #1219. It wasn't merged yet since I currently don't want to merge the other commit in the PR, and I haven't had time to try go 1.10 yet. I'll cherry-pick the fix and see if it helps.

**Comment by Ikke** [comment]
Thanks, will test this once 0.17.4 is released.
